src/lib/y2network/clients/auto.rb

```diff
@@ -89,6 +89,8 @@ def import(profile)
 
         Yast::Lan.Import(modified_profile)
 
+        update_etc_hosts
```

Comment:
I thought we agreed on doing it only in case of a Second stage but not as a general approach.

Reply:
yeah ... just checking whether you pay attention ... just joking ... I don't know where it disappeared, seems like a copy paste issue ... fixing it.